mono/mini/mini.h

```diff
@@ -3191,4 +3191,10 @@ void mono_exception_native_unwind (void *ctx, MONO_SIG_HANDLER_INFO_TYPE *info);
  */
 void mono_interruption_checkpoint_from_trampoline (void);
 
+/*
+ * Get the name of a wrapper
+ */
+const char*
+get_wrapper_type_name (int type)
```

Comment:
Github isn't showing the update, but this has become 

mini_get_wrapper_type_name